app/controllers/statistics_controller.rb

```diff
@@ -34,6 +38,24 @@ def index
 
 private
 
+  def redirect_to_research_and_statistics(format = "")
+    base_path = "#{Plek.new.website_root}/search/research-and-statistics#{format}"
+    redirect_to("#{base_path}?#{research_and_statistics_query_string}")
+  end
+
+  def research_and_statistics_query_string
+    {
+      content_store_document_type: 'published_statistics',
+      keywords: params['keywords'],
+      level_one_taxon: params['taxons'].try(:first),
```

Comment:
Should the original parameter be `taxons` or `topics`?  It's hard remembering which is used where...

Reply:
It's both!

Announcements uses topics: https://www.gov.uk/government/statistics/announcements?utf8=%E2%9C%93&topics%5B%5D=495afdb6-47be-4df1-8b38-91c8adb1eefc

Published stats uses taxons: https://www.gov.uk/government/statistics?keywords=&taxons%5B%5D=495afdb6-47be-4df1-8b38-91c8adb1eefc&departments%5B%5D=all&from_date=&to_date=

This code is on integration at the moment. If you go to either of the URLs above (on integration) you should be redirected correctly.

Reply:
Likewise, stats announcements uses "organisations", while stats uses "departments".